CS483/CS483/Kartaclysm/Data/UserConfig/ControlBindings.xml

```diff
@@ -9,7 +9,7 @@
       <driverAbility2 value="83" />
       <kartAbility1 value="68" />
       <kartAbility2 value="70" />
-      <pause value="256" />
+      <pause value="257" />
```

Comment:
GLFW is currently accepting ESC as closing the program, so the game would pause and then quit.